### ISSUE TITLE
Additional enhancements to Caller Dashboard and Team Penalty Counts dashboard

### ISSUE THREAD

**Issue by ScottChicken** [dhmay/jamstats#138]
**What version of the tools are you running, and how?**
Windows version 1.1.3

**Is your feature request related to a problem? Please describe.**
Ideas for additional enhancements to the "Caller Dashboard"

**Describe the solution you'd like**
Additional features / edits to the layout of some of the Dashboards:

- **Caller** Dashboard:
    - Team summary table: 
        - Add a "differential" row at the bottom of the table and display the difference in the score (absolute value so it doesn't matter which team is leading) - this is a pain point for those of us who are not Bobby Narco and can't do fast math in our heads
        - I would vote for removing the Calloff, NoInitial, and Skaters Played columns from the table on this board - they're in the "team summary" table but don't really come up during the game play call, at least not as much as penalty counts, score, and the others.
    - Skater table: 
        - Color skater rows based on their penalty status: when a skater hits 6 penalties color the text red (or amber or some other indicator for these...idea is to highlight skaters who could foul out in that jam)

- **Team Penalty Counts** dashboard:
     - Add the penalty totals at the top above the chart, maybe (so Team 1: 10 Team 2: 15)
     - Can you add the count # into the bars? That would make it easier than having to correlate to the axis at the bottom
     - Would LOVE a table (or another dashboard) that has penalty counts by skater by team - basically just a re-post of the penalty tracker sheet. This could also use the color highlighting I mentioned above in the Caller Dashboard, maybe going with amber for 5 penalties and red for 6 (Rat uses that color combo in their in-house Excel penalty board)

**Comment by dhmay** [comment]
Most of these are pretty achievable. This one:

`Can you add the count # into the bars? That would make it easier than having to correlate to the axis at the bottom`

has already been done in my last two builds:
<img width="793" alt="image" src="https://user-images.githubusercontent.com/714077/229373449-e142aee6-f309-4e92-a7c1-f9508fadff94.png">

Does that layout work for you?

**Comment by ScottChicken** [comment]
Ah, yeah, didn't see that one - I was looking at the Team Penalty Counts board - looks like this:
![image](https://user-images.githubusercontent.com/129686057/229375346-af404e0f-3da3-40e9-9acb-ecedf91a6185.png)
Kind of a summary for the team as a whole rather than for the skaters.

**Comment by dhmay** [comment]
Implemented: differential row, removing columns, penalty counts on skater plot.

**Comment by bullseye555** [comment]
CRG colours rows Yellow for 5, Amber for 6, and Red for fouled out [or expelled] (it actually looks at the Foul Out number which is in the JSON somewhere in the rulesets and uses that -2 for Yellow, -1 for Amber and -0 for Red)

So I'd suggest using that logic for consistency

**Comment by dhmay** [comment]
Do you want a numeric penalty count for each skater, too? I shortened some column names, so there's a little more room for that:
<img width="841" alt="image" src="https://user-images.githubusercontent.com/714077/229380533-cfd88d7a-be55-4023-9551-024f08d76d08.png">

**Comment by bullseye555** [comment]
IMO, that would be useful - so yes

**Comment by dhmay** [comment]
That'll make it easier -- I'll just color the penalty count, so it'll be clear what the color refers to.

**Comment by dhmay** [comment]
Implemented: team penalty count number on team penalty plot:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/714077/229381965-64bd45ad-a653-47e2-9913-2230ea9b8848.png">
Build: https://github.com/dhmay/jamstats/actions/runs/4591107051

**Comment by dhmay** [comment]
I added colored penalty counts to the skater table in the dashboard:
https://github.com/dhmay/jamstats/actions/runs/4591205510
It's getting a bit crowded, now. 

That covers all the enhancements you requested, @ScottChicken , except the new UI to count penalties per skater. Can I ask does the "[Team X] Skaters" plot already do that for you? The only thing that doesn't give you that you asked for, I think, is color-coding on the counts. But the top-penalty skaters are at the top, and their penalty count is given.

Closing. Reopen if you'd like any more tweaks!
